### PR TITLE
Hyper v vhd snapshot update

### PR DESCRIPTION
**Pull request by Xelu86**
# PR Summary

- Updated Hyper-V Module with current information
- Cmdlet name - Changed bolded text to code spans
- Parameters - Changed italic text to bold
- Parameters - Changed code span text to bold
- Applied text contractions
- Applied syntax reflows
- Fixed indented code blocks

## PR Checklist

- [X] **Descriptive Title:** This PR's title is a synopsis of the changes it proposes.
- [X] **Summary:** This PR's summary describes the scope and intent of the change.
- [X] **Contributor's Guide:** I have read the [contributors guide][contrib].
- [X] **Style:** This PR adheres to the [style guide][style].

### CODE REVIEW

File: docset/winserver2022-ps/hyper-v/Get-VHDSnapshot.md [hunk 1]
```diff
@@ -16,35 +16,42 @@ Gets information about a checkpoint in a VHD set.
...
-```
-PS C:\> Get-VHDSnapshot -Path "Data01.vhds" -SnapshotId 6c87351a-a39a-4581-b231-6d693b26485d
+
+```powershell
+Get-VHDSnapshot -Path "Data01.vhds" -SnapshotId 6c87351a-a39a-4581-b231-6d693b26485d
 ```
 
-This command gets information about the checkpoint that has the specified ID from the VHD set file named Data01.vhds in the current working folder.
+This command gets information about the checkpoint that has the specified ID from the VHD set file
+named Data01.vhds in the current working folder.
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/hyper-v/Get-VHDSnapshot.md:40]:
```suggestion
named `Data01.vhds` in the current working folder.
```

File: docset/winserver2022-ps/hyper-v/Get-VHDSnapshot.md [hunk 2]
```diff
@@ -54,15 +61,15 @@ Accept wildcard characters: False
...
 
 ### -ComputerName
-Specifies one or more Hyper-V hosts that run this command.
-NetBIOS names, IP addresses, and fully qualified domain names are allowable.
-The default is the local computer.
-Use localhost or a dot (.) to specify the local computer explicitly.
+
+Specifies one or more Hyper-V hosts that run this command. NetBIOS names, IP addresses, and fully
+qualified domain names are allowable. The default is the local computer. Use localhost or a dot (.)
+to specify the local computer explicitly.
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/hyper-v/Get-VHDSnapshot.md:67]:
```suggestion
qualified domain names are allowed. The default is the local computer. Use `localhost` or a dot
(`.`) to specify the local computer explicitly.
```

File: docset/winserver2022-ps/hyper-v/Remove-VHDSnapshot.md [hunk 3]
```diff
@@ -16,42 +16,48 @@ Removes a checkpoint from a VHD set file.
...
 ```
 
 ### VHDSnapshot
+
 ```
 Remove-VHDSnapshot [-PersistReferencePoint] [-VHDSnapshot] <VHDSnapshotInfo[]> [-AsJob]
- [-CimSession <CimSession[]>] [-ComputerName <String[]>] [-Credential <PSCredential[]>] [-WhatIf] [-Confirm]
- [<CommonParameters>]
+ [-CimSession <CimSession[]>] [-ComputerName <String[]>] [-Credential <PSCredential[]>] [-WhatIf]
+ [-Confirm] [<CommonParameters>]
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/hyper-v/Remove-VHDSnapshot.md:31]:
```suggestion
 [-CimSession <CimSession[]>] [-ComputerName <String[]>] [-Credential <PSCredential[]>]
 [-WhatIf] [-Confirm] [<CommonParameters>]
```

File: docset/winserver2022-ps/hyper-v/Remove-VHDSnapshot.md [hunk 4]
```diff
@@ -16,42 +16,48 @@ Removes a checkpoint from a VHD set file.
 ## SYNTAX
 
 ### Path (Default)
+
 ```
 Remove-VHDSnapshot [-Path] <String[]> [-PersistReferencePoint] -SnapshotId <Guid[]> [-AsJob]
- [-CimSession <CimSession[]>] [-ComputerName <String[]>] [-Credential <PSCredential[]>] [-WhatIf] [-Confirm]
- [<CommonParameters>]
+ [-CimSession <CimSession[]>] [-ComputerName <String[]>] [-Credential <PSCredential[]>] [-WhatIf]
+ [-Confirm] [<CommonParameters>]
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/hyper-v/Remove-VHDSnapshot.md:23]:
```suggestion
 [-CimSession <CimSession[]>] [-ComputerName <String[]>] [-Credential <PSCredential[]>]
 [-WhatIf] [-Confirm] [<CommonParameters>]
```

File: docset/winserver2022-ps/hyper-v/Remove-VHDSnapshot.md [hunk 5]
```diff
@@ -78,15 +86,15 @@ Accept wildcard characters: False
...
 
 ### -ComputerName
-Specifies one or more Hyper-V hosts that run this cmdlet.
-NetBIOS names, IP addresses, and fully qualified domain names are allowable.
-The default is the local computer.
-Use localhost or a dot (.) to specify the local computer explicitly.
+
+Specifies one or more Hyper-V hosts that run this cmdlet. NetBIOS names, IP addresses, and fully
+qualified domain names are allowable. The default is the local computer. Use localhost or a dot (.)
+to specify the local computer explicitly.
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/hyper-v/Remove-VHDSnapshot.md:92]:
```suggestion
qualified domain names are allowed. The default is the local computer. Use `localhost` or a dot
(`.`) to specify the local computer explicitly.
```

**Review by michaeltlombardi — CHANGES_REQUESTED** [review]
A few minor suggestions for syntax, but otherwise LGTM!

File: docset/winserver2022-ps/hyper-v/Get-VHDSnapshot.md (hunk 1 above)
Comment (Xelu86) [reply docset/winserver2022-ps/hyper-v/Get-VHDSnapshot.md:40]: Updated

**Review by michaeltlombardi — APPROVED** [review]

**Pull request merged**